code/_onclick/hud/picture_in_picture.dm

```diff
@@ -1,3 +1,5 @@
+#define MAX_DIMENSIONS 10
```

Comment:
this is used in only a single place, just set the value and comment in the proc instead.